src/labels/CustomLabels.labels

```diff
@@ -3046,7 +3046,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>hhCmpDedupeNoReport</shortDescription>
-        <value>Could not find report. Create a report with the Report Unique Name of  &apos;NPSP_Campaign_Household_Mailing_List&apos;. NOTE: Report type should be Campaigns with Contacts. Filter for Member Status does not contain &apos;Duplicate&apos;.</value>
```

Comment:
@smarovich and @judisohn Change to the label used to display an error when the report cannot be found

Reply:
@force2b I'd make some very minor changes: 
Could not find the report. Create a report and save it with the **Report Unique Name** of  &apos;NPSP_Campaign_Household_Mailing_List&apos;. Use the Campaigns with Contacts report type and add a filter for Member Status does not contain &apos;Duplicate&apos;.

Reply:
@smarovich Good suggestion, though the report unique name as to be NPSP_Campaign_Household_Mailing_List_V2. I'm open to changing the report unique name or label, but it has to be different than the previous report's unique name.

Reply:
Ah ok. That report name is fine with me. I think we should keep it similar to the original name.  

Could not find the report. Create a report and save it with the Report Unique Name of 'NPSP_Campaign_Household_Mailing_List_V2'. Use the Campaigns with Contacts report type and add a filter for Member Status does not contain 'Duplicate'.

Reply:
Thanks, though one more item. I had added a fragment for "CampaignId equals blank and" as part of the filters. This is a new requirement for the report filtering for it to work correctly (see below). Can I put that back in so it reads as follows?

`Could not find the report. Create a report and save it with the Report Unique Name of 'NPSP_Campaign_Household_Mailing_List_V2'. Use the Campaigns with Contacts report type and add filters for CampaignId equals blank and Member Status does not contain 'Duplicate'.`

Reply:
That looks great!